holmes/plugins/toolsets/newrelic.py

```
@@ -117,9 +115,7 @@
             toolset=toolset,
         )
 
-    def _invoke(
-        self, params: dict, user_approved: bool = False
-    ) -> StructuredToolResult:
+    def _invoke(self, params: Any, user_approved: bool = False) -> StructuredToolResult:
```

Ruff (0.12.2)

Unused method argument: `user_approved`

(ARG002)